CMakeLists.txt

```diff
@@ -300,6 +301,24 @@ endif()
 
 add_subdirectory(capi)
 
+#-----------------------------------------------------------------------------
+# Setup build directory for targets: output directories of
+# library, archive and runtime.
+# Place executables and shared libraries in the same location for
+# convenience of direct execution from common spot and for
+# convenience in environments without RPATH support.
+#-----------------------------------------------------------------------------
+set_target_properties(${_targets}
```

Comment:
This is fine, I just mean dispense with the GEOS_ADD_TARGET machinery, and just call `set_target_properties(geos geos_c ... )` without the indirection of the _targets variables.

Reply:
Yeah, fine by me. I'll delete `GEOS_ADD_TARGET` for now.